core/runner_actor.py

```diff
@@ -0,0 +1,102 @@
+# Add the parent directory to the PATH to allow imports.
```

Comment:
Why do we need lines 1-5? I think this is only useful for things inside `tests`. We're not running files directly and calling their main entrypoint anymore.

Reply:
Copied them exactly from `DMLRunner` and accidentally left them in. Removed.